tests/utils.py

```diff
@@ -136,6 +136,8 @@ def default_config(name):
     config.rc_messages_per_second = 10000
     config.rc_message_burst_count = 10000
 
+    config.use_frozen_dicts = False
```

Comment:
I'm wondering if this is actually used (since afaict `use_frozen_dicts` is only read by code which is never run under the UTs), and if not I would probably choose to omit this. But I'm not that bothered and this needs merging.